RequestManagementSystem/Client/Request.py

```diff
@@ -370,7 +370,7 @@ def Status( self, value ):
     # If the status moved to Failed or Done, update the lastUpdate time
     if value in ( 'Done', 'Failed' ):
       if value != self._Status:
-        self.LastUpdate = datetime.datetime.utcnow().replace( microsecond = 0 )
```

Comment:
However, it was a acknowledged bug that was fixed, so it should not happen anymore.... http://www.logilab.org/revision/90049